docs/how-to-guides/contributing.md

```diff
@@ -0,0 +1,33 @@
+This document explains the processes and practices recommended for contributing enhancements to the Indico operator.
+
+* Generally, before developing enhancements to this charm, you should consider [opening an issue ](https://github.com/canonical/discourse-k8s-operator/issues)explaining your use case.
```

Comment:
is the space inside the link `[... ]` deliberate?

Reply:
Good catch. We'll problably have to double check the rest of the docs for this.